{"uuid": "502e21e6-74cb-4188-b597-8c7e552ab15f", "children": ["f97d9e6a-09cd-4958-8fc8-a17625d99c8e", "a088e0a9-dbb8-4065-926b-5b3d77234269", "389e22ce-f73e-4db5-9149-43080694b7bd", "acfc6aac-00a4-431b-acf2-f342f90ed150", "e6f4f4fa-98a3-46f2-9d9c-3402cc7aeed7"], "befores": [{"name": "_Class__pytest_setup_class", "status": "passed", "start": 1586835816834, "stop": 1586835816834}], "afters": [{"name": "_Class__pytest_setup_class::0", "status": "broken", "statusDetails": {"message": "NameError: name 'mail_pass' is not defined\n", "trace": "  File \"/usr/local/lib/python3.7/site-packages/allure_commons/_allure.py\", line 185, in __call__\n    return self._fixture_function(*args, **kwargs)\n  File \"/usr/local/lib/python3.7/site-packages/_pytest/fixtures.py\", line 800, in _teardown_yield_fixture\n    next(it)\n  File \"/usr/local/lib/python3.7/site-packages/_pytest/python.py\", line 707, in xunit_setup_class_fixture\n    _call_with_optional_argument(func, self.obj)\n  File \"/usr/local/lib/python3.7/site-packages/_pytest/python.py\", line 634, in _call_with_optional_argument\n    func(arg)\n  File \"/Users/sonic/Project/Python/Pytest/scripts/test_case.py\", line 41, in teardown_class\n    EmailHandler().send_email()\n  File \"/Users/sonic/Project/Python/Pytest/uti/EmailHandler.py\", line 59, in send_email\n    smtp_obj.login(mail_user, mail_pass)\n"}, "start": 1586835816983, "stop": 1586835822889}], "start": 1586835816834, "stop": 1586835822890}